Specs/DateTime/Portuguese/TimeExtractor.json

```diff
@@ -564,26 +564,26 @@
     ]
   },
   {
-    "Input": "Voltarei de madrugada",
+    "Input": "Voltarei no meio da madrugada",
```

Comment:
Does madrugada by itself now return a time range, as in Spanish?

Reply:
yes, added test case in Portuguese DateTimeModel.